stream-chat-android-client/src/test/java/io/getstream/chat/android/client/parser2/UpstreamChannelDtoAdapterTest.kt

```diff
@@ -0,0 +1,31 @@
+package io.getstream.chat.android.client.parser2
+
+import io.getstream.chat.android.client.api2.model.dto.UpstreamChannelDto
+import io.getstream.chat.android.client.parser2.testdata.ChannelDtoTestData
+import org.assertj.core.api.Assertions
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.assertThrows
+
+internal class UpstreamChannelDtoAdapterTest {
+
+    private val parser = MoshiChatParser()
+
+    @Test
+    fun `Serialize JSON channel with custom fields`() {
+        val jsonString = parser.toJson(ChannelDtoTestData.upstreamChannel)
+        Assertions.assertThat(jsonString).isEqualTo(ChannelDtoTestData.upstreamJson)
```

Comment:
Nit: we could import `assertThat` directly, so that we don't have `Assertions.` at the front of these kinds of lines in the newly added tests